command.go

```diff
@@ -1298,6 +1310,10 @@ func (c *Command) updateParentsPflags() {
 		c.parentsPflags.SortFlags = false
 	}
 
+	if c.globNormFunc != nil {
```

Comment:
I think, it's better not to check this. User may remove global normalization func with `cmd.SetGlobalNormalizeFunc(nil)` and it will not work.

Reply:
Ah yes you're right: for some reason I thought you could not pass `nil` to `SetGlobalNormalizeFunc`.
But then I think this check should also be removed from `Command.AddCommand`: otherwise you may end up in a situation where the parent command does not have a normalisation function set, but the child does, which contradicts the assumption that it is set consistently throughout the command tree.

Reply:
I was wrong. It's better to check if c.globNormFunc is nil. If user wants to remove globNormFunc, it's better to use `func(f *FlagSet, name string) NormalizedName { return NormalizedName(name) }`, because using nil funcs can cause to nil-pointer errors.